### DISCUSSION TITLE
Is the app unmaintained?

### DISCUSSION THREAD

**Discussion by AuroraMartell** [KDE/ghostwriter#881] [General]
There hasn't been a new release since 2022. I used to use the app but switched to Mac (where's there's no equivalent). Now that I'm back on Windows 11, the app hasn't been updated. Any news/ideas/alternatives?

**Comment by abenson** [comment]
[New version last week](https://github.com/KDE/ghostwriter/releases/tag/v24.02.2).

Releases seem to have changed since the move to KDE, but they're still happening.

~~[Official macOS guidance](https://ghostwriter.kde.org/download/#macos)~~ Misread your statement.

**Reply by AuroraMartell** [reply]
Any ETAs on a new Windows version (that's not portable)?

**Comment by wereturtle** [comment]
Actually, I’m looking for volunteers to get a KDE Craft configuration for Windows going. Are you interested?

-- 
  Megan
  ghostwriter developer

**Comment by AuroraMartell** [comment]
> Actually, I’m looking for volunteers to get a KDE Craft configuration for Windows going. Are you interested?
> […](#)
> -- Megan ghostwriter developer
> On Fri, Apr 19, 2024, at 7:56 AM, Aurora Martell wrote: Any ETAs on a new Windows version (that's not portable)? — Reply to this email directly, view it on GitHub <[#881 (reply in thread)](https://github.com/KDE/ghostwriter/discussions/881#discussioncomment-9167747)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACPFAAXPVEPF6KG26MBAA7LY6EWCRAVCNFSM6AAAAABGPIYKU2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNRXG42DO>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Can't code, sorry.
